src/components/Rows.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
```

Comment:
- [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉